lib/dor/services/client/release_tags.rb

```diff
@@ -30,12 +30,14 @@ def create(tag:)
 
         # List release tags for an object
         #
+        # @param public [Boolean] indicates if we only want public release tags (defaults to false)
         # @raise [NotFoundResponse] when the response is a 404 (object not found)
         # @raise [UnexpectedResponse] if the request is unsuccessful.
         # @return [Array<ReleaseTag>]
-        def list
+        def list(public: false)
+          params = public ? 'public=true' : ''
           resp = connection.get do |req|
```

Comment:
We're fairly sloppy about this in our clients, but this would be cleaner:
```
resp = connection.get(url, {public:})
```

Reply:
made suggested change and updated specs (however, rubocop is currently targeting ruby 3.0, so had to make a small tweak)

if we want to target ruby 3.2 in rubocop, let's do that separately, as autocorrect will make a bunch of changes